recipes/demquery/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: demquery
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/28/dc/119312bc384c4a6526fdc0e4d655552516d812147c890539f92ad8924445/demquery-{{ version }}.tar.gz
```

Could you please change it to:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```